README.md

```diff
@@ -25,7 +25,7 @@ Library compatibility with Exonum core:
 
 | JavaScript light client | Exonum core |
 |---|---|
-| 0.18.1 | 0.13.* |
+| 0.18.2 | 0.13.* |
```

Comment:
Technically, this version is compatible with some core revision *after* `0.13.0-rc.2` (it's possible to find the revision by checking `integration-tests/Cargo.toml`). As we've discussed personally, there is at least one breaking change since 0.13.0-rc.2: the serialization of `Block`.